src/CCPNSimulatorPlugin.js

```diff
@@ -1,13 +1,25 @@
 const xmlBuilder = require('xmlbuilder');
 const xmlParser = require('fast-xml-parser');
+const AbstractPlugin =
+    require('renew-simulator-gateway/src/plugin/AbstractPlugin');
```

Comment:
Why are you using the require statement instead fo the import? I would prefer the import.

Reply:
That would require using the `--experimental-modules` or a transpiler  wouldn't it?

Reply:
whats the case in not using a transpiler?